README.md

```diff
@@ -1,2 +1,43 @@
 # lux-benchmark
 Experiments to benchmark Lux Performance
+
+## Setup
+First, ensure `lux-datasets` is in the same directory as this one locally. This will allow you to run the following command to generate large airbnb and communities datasets:
+
+```
+python make_dataset.py
+```
+
+## Experiments
+
+### Overall Benchmarking
+The time used for benchmarking can be broken down into three parts:
+
+- t_meta: time of computing the metadata
+- t_recs: time for computing and generating the visualization recommendations 
+- t_render: time for renderer (e.g., Matplotlib, Altair) to build the visualization in the backend
+
+As an aside, `t_render` is a good representation of the runtime on the front-end, since after the visualizations are rendered, it takes very little time to print the visualizations.
```

Comment:
Is this actually true? Since scatterplot would take very little time on the backend, but lots of time to render on the frontend.

Reply:
Oh I must've misunderstood what you meant when I asked about this via Slack. The overall benchmarking also tracks `t_print` which in the results is negligible. I assumed `t_print` was the front end showing the rendered visualization in the notebook, and since it's rendered already it doesn't take any time. I guess, would it actually be a lot more if we were actually running it in a notebook? The script doesn't show the visualization, so that would be the reason why `t_print` is so small?

Reply:
Yeah `t_print` just tracks the cost of print(df) on the backend. It doesn't involve any of the frontend rendering (the process ends when the JSON representation of the widget is generated, so no frontend code is actually ran).